cardboard/settings.py

```diff
@@ -311,6 +311,10 @@ class TaskPriority(Enum):
         },
     },
     "loggers": {
+        "": {
```

Comment:
it looks like you can alternatively pull this out to the top level in a `root` entry a la https://docs.djangoproject.com/en/5.0/topics/logging/#id3 

don't think there's any functional difference between the two

Reply:
Yep. Done.